pages/index.js

```diff
@@ -47,8 +48,9 @@ export default function Home({ date, expirationDate }) {
       )}
       <WarningText>
         Please note that applications for the Discretionary Business Grant close
-        at 23:59 on Friday 28th August 2020. Any applications submitted after
-        this time will be invalid.
+        at {dateFormat(new Date(expirationDate), 'h:MM:ss TT')} on{' '}
+        {dateFormat(new Date(expirationDate), 'dddd, mmmm dS yyyy')}. Any
```

Comment:
Not a big deal but probably we don't need a 3rd party dependency https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleDateString